iosapp/controller/controller.entitlements

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
   <dict>
+    <key>com.apple.developer.family-controls</key>
```

Comment:
required to get the `ManagedSettingsStore` to do anything from the controller extension target